### PR TITLE
ghostscript 9.55.0

### PR DESCRIPTION
**Pull request by miccal**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

**Comment by miccal** [comment]
Not sure what to do here:
```
Undefined symbols for architecture arm64:
  "_gscms_transformm_color_const", referenced from:
      _gscms_transform_color in gsicc_lcms2.o
ld: symbol(s) not found for architecture arm64
```
```
Undefined symbols for architecture x86_64:
  "_gscms_transformm_color_const", referenced from:
      _gscms_transform_color in gsicc_lcms2.o
ld: symbol(s) not found for architecture x86_64
```
```
/home/linuxbrew/.linuxbrew/bin/ld: ./obj/gsicc_lcms2.o: in function `gscms_transform_color':
gsicc_lcms2.c:(.text+0x693): undefined reference to `gscms_transformm_color_const'
```

**Comment by iMichka** [comment]
Patch can be found here: http://git.ghostscript.com/?p=ghostpdl.git;a=patch;h=830afae5454dea3bff903869d82022306890a96c

**Comment by SMillerDev** [comment]
Looks like we've been carrying a patch for nothing 🥳 .

**Review by SMillerDev — APPROVED** [review]

File: Formula/ghostscript.rb [hunk 1]
```diff
@@ -59,6 +55,11 @@ class Ghostscript < Formula
     sha256 "0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401"
   end
 
+  patch do
+    url "http://git.ghostscript.com/?p=ghostpdl.git;a=patch;h=830afae5454dea3bff903869d82022306890a96c"
+    sha256 "b0b7a042e50f3eb5f7de8f8e833397953f8730030c71572ae372f44f7144c9bb"
+  end
```
Comment (carlocab) [inline Formula/ghostscript.rb:61]: What was this patch for? It no longer produces the same checksum:
```
❯ curl -sL http://git.ghostscript.com/\?p\=ghostpdl.git\;a\=patch\;h\=830afae5454dea3bff903869d82022306890a96c | sha256sum
9c282e3f732b5c182ab761e187850d9f4c29d938a93e3785d9cb6ced13111d6c  -
```

Comment (miccal) [reply Formula/ghostscript.rb:61]: I do not think I actually added this myself -- I believe @SMillerDev added this?

**Pull request closed without merging**